src/get_hepdata.py

```diff
@@ -67,54 +72,55 @@
     error_label = ''
     first_entry = yaml_data['dependent_variables'][0]['values'][0]
     for err in first_entry['errors']:
-        error_label += err['label']+'_lo ' + err['label']+'_hi '
+      error_label += err['label']+'_lo ' + err['label']+'_hi '
     data_header +='# Label xmin xmax y ' + error_label + '\n'
-    
-#    print(yaml_data['independent_variables'][0]['values'])
+      
+    # print(yaml_data['independent_variables'][0]['values'])
     for x in yaml_data['independent_variables'][0]['values']:
-        xlo.append(x['low'])
-        xhi.append(x['high'])                    
-#        print(x['low'],x['high'])
+      xlo.append(x['low'])
+      xhi.append(x['high'])
+      # print(x['low'],x['high'])
 
-#    print(yaml_data['dependent_variables'][0]['values'])
+    # print(yaml_data['dependent_variables'][0]['values'])
     for v in yaml_data['dependent_variables'][0]['values']:
-#        print(v['value'])
-        yval.append(v['value'])
-
-        errs.append('')
-        for err in (v['errors']):
-#            print(err['label'])
-            try:
-                e = str(err['symerror'])
-                # Convert if percentage
-#                if (e[-1]=='%'):
-#                    e = str(float(e[:-1])*v['value']*0.01)
-                errs[-1] += ' ' + e + ' ' + e
-#                print ('  sym',e,e)
-            except KeyError:
-                e = err['asymerror']
-                errs[-1] += ' ' + str(e['plus']) + ' ' + str(e['minus'])
-#                print ('  asym',e['plus'],e['minus'])
+      # print(v['value'])
+      yval.append(v['value'])
+
+      errs.append('')
+      for err in (v['errors']):
+        # print(err['label'])
+        try:
+          e = str(err['symerror'])
+          # Convert if percentage
+          # if (e[-1]=='%'):
+          #   e = str(float(e[:-1])*v['value']*0.01)
```

Comment:
Let's enable this part for now - it's perhaps not the best implementation but would avoid the atlas error you observed